### ISSUE TITLE
Program Error: "Instruction #5 Failed"

### ISSUE THREAD

**Issue by PomegranateApps** [metaplex-foundation/metaplex#2256]
I am getting this error when I attempt to mint an NFT on my candy machine.

`Program Error: "Instruction #5 Failed"`

https://explorer.solana.com/tx/422juo3RTsXwQysyVEZZpVNhoVi84uifPxTjUQwDJtvbMdbQ6ocyH9gx6KaJT9M4f4ZEGut5hGciYS6cKSudgRCQ?cluster=devnet

Where can I find documentation on this error message?

**Comment by polarysfoundation** [comment]
Hello buddy, could you tell me what you are using to mint?, I mean, do you use a third party mint page?, what is your source code, since I have the same problem and if we have the same dependencies, we could find what causes the mistake.

**Comment by PomegranateApps** [comment]
It turned out the wallet was too low on funds. There was enough to mint (there was around ~0.8 SOL remaining), but there seems to be a minimum required SOL to have in the wallet to mint. I'm not sure if it's possible, but it would be great to have more descriptive errors regarding this issue.

**Comment by polarysfoundation** [comment]
Yes, I suspected it yesterday by reviewing the transactions coming from the wallet that you placed in the report, I suppose that the error is caused by not having enough funds. and since there are not many details regarding the error, sometimes it is a headache. Thanks a lot.

**Comment by aUsABuisnessman** [comment]
> Hello buddy, could you tell me what you are using to mint?, I mean, do you use a third party mint page?, what is your source code, since I have the same problem and if we have the same dependencies, we could find what causes the mistake.
> 

is Polarys a progressive
